lib/global-error/src/error.rs

```diff
@@ -8,7 +8,7 @@ use types::rivet::chirp;
 
 pub type GlobalResult<T> = Result<T, GlobalError>;
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Why is this needed?

Reply:
Hmm from my perspective, something generic like this probably should implement clone unless there's a reason it shouldn't. Is there something that comes to mind for why it shouldn't?